templates/upload.html

```diff
@@ -43,20 +58,22 @@ <h3>Upload Syllabus</h3>
       <b>Please be aware of any policies your school may have about the sharing of syllabi.<br> By uploading a syllabus, you certify that you have done this and there is no problem with it.</b>
     </div>
   </div>
-  <div class="form-group">
-    <label for="file"><b>Syllabus (PDFs only)</b></label>
-    <input class="form-control-file" id="file" type="file" name="file" accept="application/pdf" required>
+
+  <div class="form-row">
+    <div class="form-group col-md-4">
+      <label for="file"><b>Syllabus (PDFs only)</b></label>
+      <input class="form-control-file" id="file" type="file" name="file" accept="application/pdf" required>
+    </div>
+    <div class="form-group col-md">
+      <div id="message" class="alert d-inline-block warning" style="display: none !important;" role="alert"> </div>
```

Comment:
Question: since there's nothing in here by default, would it even display anything if you removed the `display: none`?

Reply:
Yep, removing that would display a nice red block that houses no text by default